docs/design.md

```diff
@@ -342,22 +342,22 @@ Transactions are managed by the client proxy (or gateway in SQL Azure
 parlance). Unlike in Spanner, writes are not buffered but are sent
 directly to all implicated ranges. This allows the transaction to abort
 quickly if it encounters a write conflict. The client proxy keeps track
-of all written keys in order to cleanup write intents upon transaction
-completion.
-
+of all written keys in order to post-process write intents upon completion.
 If a transaction is completed successfully, all intents are upgraded to
 committed. In the event a transaction is aborted, all written intents
-are deleted. The client proxy doesn’t guarantee it will cleanup intents;
-but dangling intents are upgraded or deleted when encountered by future
-readers and writers and the system does not depend on their timely
-cleanup for correctness.
+are deleted.
 
+The client proxy doesn’t guarantee it will post-process intents.
```

Comment:
I don't think we ever called this post-processing. Use the term "resolving" instead.